ncl/ncl_index/ncl-index-table.csv

```diff
@@ -1,3 +1,4 @@
 NCL Function, Description, Python Equivalent, Notes
 `days_in_month <https://www.ncl.ucar.edu/Document/Functions/Built-in/days_in_month.shtml>`__,"Calculates the number of days in a month given month and year","``cftime.datetime().daysinmonth``",`details <../ncl_entries/days_in_month.ipynb>`__
 `day_of_week <https://www.ncl.ucar.edu/Document/Functions/Built-in/day_of_week.shtml>`__,"Calculates the day of the week given month, day, and year","``cftime.datetime().dayofwk``",`details <../ncl_entries/day_of_week.ipynb>`__
+`spectral analysis <https://www.ncl.ucar.edu/Applications/spec.shtml>`__," Calculates spectra of a series.","``scipy.signal.periodogram()``, ``scipy.signal.csd()`, or ``scipy.signal.coherence()``",`example notebook <../receipts/NCL_spectral_analysis.ipynb>`__
```

Comment:
Currently this link is broken as a `404`, it seems like the site can't find this directory

Reply:
`NCL_spectral_analysis.ipynb` is currently living in `ncl` (that is where it is being [generated](https://ncar.github.io/geocat-applications/_preview/39/ncl/NCL_spectral_analysis.html)), should it be within `ncl/receipts`?

Reply:
No sorry, that was from before I had a separate NCL and NCL-receipts notebook.